src/main/java/org/duckstudy/model/lotto/constant/WinningRank.java

```diff
@@ -0,0 +1,45 @@
+package org.duckstudy.model.lotto.constant;
+
+import java.util.stream.Stream;
+
+public enum WinningRank {
+
+    NONE(0, 0, 0),
+    FIFTH(3, 5000, 3),
```

Comment:
```suggestion
    FIFTH(3, 5_000, 3),
```
3자리가 넘어갈때는 `_`로 가독성을 올릴 수 있어요.

Reply:
오 그렇군요! 😀 반영하겠습니다 !